src/android/AudioPlayer.java

```diff
@@ -645,9 +664,16 @@ private boolean readyPlayer(String file) {
                     if (file!=null && this.audioFile.compareTo(file) == 0) {
                         //maybe it was recording?
                         if (player == null) {
+                            audioManager.setSpeakerphoneOn(false);
```

Comment:
shall be?
```suggestion
                            audioManager.setSpeakerphoneOn(isSpeakerphone);
```

and I think it would be good to create method `setOutput` and call it in that two places

Reply:
changed